src/odemis/driver/pigcs.py

```diff
@@ -805,9 +804,13 @@ def GetMotionStatus(self, check=True):
         mv_axes = set()
         while bitmap > 0:
             if bitmap & 1:
-                mv_axes.add(i)
+                mv_axes.add(bytes(i))
             i += 1
             bitmap >>= 1
+        # if axes labels are not digits convert the mv_axes to characters.
+        if not all(k.isdigit() for k in self._channels):
+            # Assume axes are sorted alphabetically, i.e. 1 corresponds to x, 2 to y 3 to z.
+            mv_axes = set(sorted(self._channels)[:int(max(mv_axes))])
```

Comment:
I think a short and more correct way would be:
`set(n for i, n in enumerate(sorted(self._channels)) if bytes(i + 1) in mv_axes)`